XML/TestCases/BVT.xml

```diff
@@ -230,12 +230,15 @@
         <Tags>core</Tags>
     </test>
     <test>
-        <testName>BVT-CORE-RELOAD-MODULES</testName>
+        <testName>BVT-CORE-RELOAD-MODULES-SMP</testName>
+        <setupScript>.\TestScripts\Windows\SETUP-Change-CPU.ps1</setupScript>
         <testScript>BVT-CORE-RELOAD-MODULES.ps1</testScript>
         <files>.\TestScripts\Linux\BVT-CORE-RELOAD-MODULES.sh,.\TestScripts\Linux\utils.sh</files>
-        <setupType>SingleVM</setupType>
+        <setupType>Stress</setupType>

```

Comment:
why change this?

Reply:
This is how we handle multiple vCPU on Azure. On Hyper-V we use setupscripts but on Azure we use SetupType tags to point what type of VM do we want